srtp/srtp.c

```diff
@@ -676,8 +676,8 @@ static srtp_err_status_t srtp_kdf_generate(srtp_kdf_t *kdf, srtp_prf_label label
 }
 
 static srtp_err_status_t srtp_kdf_clear(srtp_kdf_t *kdf) {
-    memset(kdf->master_key, 0x0, MAX_SRTP_AESKEY_LEN);
-    memset(kdf->master_salt, 0x0, MAX_SRTP_SALT_LEN);
+    octet_string_set_to_zero(kdf->master_key, 0x0, MAX_SRTP_AESKEY_LEN);
```

Comment:
This function does not need the 0x0 parameter in the middle. Function signature is:

~~~
octet_string_set_to_zero(void *s, size_t len);
~~~